llvm/tools/llc/llc.cpp

```diff
@@ -186,6 +186,12 @@ static cl::opt<std::string> RemarksFormat(
     cl::desc("The format used for serializing remarks (default: YAML)"),
     cl::value_desc("format"), cl::init("yaml"));
 
+static cl::opt<bool> TryUseNewDbgInfoFormat("try-experimental-debuginfo-iterators",
+    cl::desc("Enable debuginfo iterator positions, if they're built in"),
+    cl::init(false));
```

Comment:
Didn't this land in another patch? (or maybe I'm confused).

Reply:
I think the previous one was for `opt`, while this is `llc`.